docker/Dockerfile.build

```diff
@@ -17,6 +20,6 @@ COPY go.sum go.sum
 RUN go mod download
 
 COPY . .
-RUN make livepeer livepeer_cli
+RUN HIGHEST_CHAIN_TAG=${HIGHEST_CHAIN_TAG} make livepeer livepeer_cli
```

Comment:
Do we need to set `HIGHEST_CHAIN_TAG` here if the `ENV` statement up top sets `HIGHEST_CHAIN_TAG` already?

Reply:
yes

Reply:
I provided detail for this item in the wrong thread :-/

The way it works as-is, with the ENV declared and also passed explicitly to the build process ensures that the env var is defined both during the build and in the running release image. It will be the only way to tell after the fact what variable was set during the build.

Reply:
TBH, I'm beginning to question it myself now.  You'd think this would be unnecessary if it's actually in the running build environment.   I looked over some projects from the past, and I've always done it this way, but I think it was actually for the sake of compatibility with older docker versions, or simply force of habit.  If you want to give it a try without setting it explicitly in the RUN line, I'm willing to give that a try :-)

Reply:
88d9e9ce6324d72a3e97b47dc5c0dda3334984e6